lib/ae_mdw/db/oracle.ex

```diff
@@ -39,6 +25,13 @@ defmodule AeMdw.Db.Oracle do
       )
   end
 
+  @spec oracle_tree!(Blocks.block_hash()) :: tuple()
+  def oracle_tree!(block_hash) do
+    block_hash
+    |> :aec_db.get_block_state()
+    |> :aec_trees.oracles()
+  end
+
```

Comment:
Can we move this function back to where it was since it wasn't changed?

Reply:
you're right, during the refactoring it was only moved.